evm/src/cpu/kernel/stack/stack_manipulation.rs

```diff
@@ -56,14 +55,17 @@ fn expand(names: Vec<StackPlaceholder>, replacements: Vec<StackReplacement>) ->
                 // May be either a named item or a label. Named items have precedence.
                 if stack_blocks.contains_key(&name) {
                     let n = *stack_blocks.get(&name).unwrap();
-                    (0..n)
-                        .map(|i| {
-                            let literal_name = format!("block_{}_{}", name, i);
-                            StackItem::NamedItem(literal_name)
-                        })
-                        .collect_vec()
-                } else if stack_names.contains(&name) {
-                    vec![StackItem::NamedItem(name)]
+                    if n == 1 {
```

Comment:
WDYT of removing this case? Seems the `else` code should be equivalent except it'd add `@` and `.0`, which I guess doesn't hurt

Reply:
Oh sure!